### ISSUE TITLE
Async and Defer is broken

### ISSUE THREAD

**Issue by Remzi1993** [Remzi1993/wp-jquery-manager#8]
Async and Defer are broken for both jQuery and jQuery Migrate.

**Comment by Remzi1993** [comment]
Will be fixed in the future. This has low priority.

**Comment by oldrup** [comment]
Hey! Thank you for creating this plugin, helps me use more modern and safe versions of jquery if at all needed on a site.

I'm not too much of the dysfunctional async/defer option though. Most caching plugins offers that functionality anyway, and as it isn't working, why not just remove the option completely? Would clean up the UI a bit.

Stay safe!
Bjarne

**Comment by cp-bwg** [comment]
> Hey! Thank you for creating this plugin, helps me use more modern and safe versions of jquery if at all needed on a site.
> 
> I'm not too much of the dysfunctional async/defer option though. Most caching plugins offers that functionality anyway, and as it isn't working, why not just remove the option completely? Would clean up the UI a bit.
> 
> Stay safe!
> Bjarne

Agreed - if something is not working at all - reference this issue as a future enhancement, but remove the options.
